src/components/Web3ReactManager/index.tsx

```diff
@@ -21,6 +21,16 @@ const Web3ReactManager = ({ children }) => {
   const history = useHistory();
   const rpcUrls = useRpcUrls();
 
+  const originalFetch = window.fetch;

```

Comment:
Can you add a comment here on what it does?.

Reply:
Done, good call